From e88bd677789f4a5a932c28bd449b50dc2c339b02 Mon Sep 17 00:00:00 2001
From: Sven Brauch <svenbrauch@googlemail.com>
Date: Wed, 16 Oct 2013 17:41:35 +0200
Subject: [PATCH] Adjust to libqinfinity installation directory change

This collection of cmake scripts is a mess, it needs to be cleaned asap;
probably all but one of them can be removed, and the libqinfinity config
script should be provided by libqinfinity itself.
---
 CMakeModules/FindLibqinfinity.cmake | 2 --
 1 file changed, 2 deletions(-)

diff --git a/CMakeModules/FindLibqinfinity.cmake b/CMakeModules/FindLibqinfinity.cmake
index 806f3d6..ea08663 100644
--- a/CMakeModules/FindLibqinfinity.cmake
+++ b/CMakeModules/FindLibqinfinity.cmake
@@ -40,8 +40,6 @@ else( LIBQINFINITY_LIBRARIES AND LIBQINFINITY_INCLUDES )
 			/usr/include
 			/usr/local/include
 			/opt/local/include
-		PATH_SUFFIXES
-			libqinfinity-1.0
 	)
 	
 	find_library( LIBQINFINITY_LIBRARY
-- 
1.8.4.rc3

